src/value/borrowed.rs

```diff
@@ -91,221 +80,251 @@ impl<'v> Value<'v> {
 impl<'v> ValueTrait for Value<'v> {
     type Key = Cow<'v, str>;
 
+    #[inline]
```

Comment:
(same question about the more verbose `inline`)

Reply:
I'm always unsure, I tend to give the `#[inline]` hint for tiny functions to push rustc/llvm more towards considering to inline them (as it's replacing 1 command with 1 command) and `#[inline(always)]` for things that we know we want inlined (based on upstream), and the rest I do not touch either and trust that rustc/llvm knows better then me :P